### ISSUE TITLE
warning " > rails-erb-loader@5.5.2" has unmet peer dependency "webpack@^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0".

### ISSUE THREAD

**Issue by Rovel** [usabilityhub/rails-erb-loader#80]
As I use `yarn install` I'm getting this warning:

```
warning " > rails-erb-loader@5.5.2" has unmet peer dependency "webpack@^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0".
```

Should I need to worry about this? I tried to upgrade/match dependencies but it still throwing this warning.

Any concerns or tips?

**Comment by PikachuEXE** [comment]
What's your `webpack` version?

**Comment by Rovel** [comment]
3.5

**Comment by PikachuEXE** [comment]
I am using 
```
webpack@^4.32.2:
  version "4.35.2"
```
But I don't see that message running `yarn install`

**Comment by PikachuEXE** [comment]
I only see a lot of warnings during `yarn upgrade`
I normally just ignore them :P
```
warning " > @babel/plugin-proposal-class-properties@7.5.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/plugin-proposal-class-properties > @babel/helper-create-class-features-plugin@7.5.0" has unmet peer dependency "@babel/core@^7.0.0".
warning " > @babel/plugin-proposal-object-rest-spread@7.5.1" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/plugin-proposal-object-rest-spread > @babel/plugin-syntax-object-rest-spread@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > @babel/plugin-syntax-dynamic-import@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > @babel/plugin-syntax-flow@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > @babel/plugin-transform-exponentiation-operator@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > @babel/plugin-transform-flow-strip-types@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > @babel/preset-env@7.5.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-syntax-json-strings@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-syntax-async-generators@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-proposal-dynamic-import@7.5.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-proposal-json-strings@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-proposal-async-generator-functions@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-proposal-optional-catch-binding@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-syntax-optional-catch-binding@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-proposal-unicode-property-regex@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-block-scoped-functions@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-block-scoping@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-computed-properties@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-classes@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-dotall-regex@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-arrow-functions@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-duplicate-keys@7.5.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-literals@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-function-name@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-member-expression-literals@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-modules-amd@7.5.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-modules-umd@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-modules-commonjs@7.5.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-named-capturing-groups-regex@7.4.5" has unmet peer dependency "@babel/core@^7.0.0".
warning "@babel/preset-env > @babel/plugin-transform-for-of@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-modules-systemjs@7.5.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-parameters@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-object-super@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-regenerator@7.4.5" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-property-literals@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-new-target@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-shorthand-properties@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-reserved-words@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-spread@7.2.2" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-sticky-regex@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-template-literals@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-typeof-symbol@7.2.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-unicode-regex@7.4.4" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-async-to-generator@7.5.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-env > @babel/plugin-transform-destructuring@7.5.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > @babel/preset-flow@7.0.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > @babel/preset-typescript@7.3.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-typescript > @babel/plugin-transform-typescript@7.5.1" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@babel/preset-typescript > @babel/plugin-transform-typescript > @babel/plugin-syntax-typescript@7.3.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > webpack-assets-manifest@3.1.1" has unmet peer dependency "webpack@>=4.4.0".
warning " > webpack-cli@3.3.5" has unmet peer dependency "webpack@4.x.x".
warning " > leaflet.markercluster@1.4.1" has incorrect peer dependency "leaflet@~1.3.1".
warning " > rails-erb-loader@5.5.2" has unmet peer dependency "webpack@^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0".
warning " > raw-loader@3.0.0" has unmet peer dependency "webpack@^4.3.0".
warning " > @typescript-eslint/parser@1.11.0" has incorrect peer dependency "eslint@^5.0.0".
warning " > webpack-dev-server@3.7.2" has unmet peer dependency "webpack@^4.0.0".
warning "webpack-dev-server > webpack-dev-middleware@3.7.0" has unmet peer dependency "webpack@^4.0.0".
```

**Comment by bragovo** [comment]
I have the same warning. Seems this PR must fix it: https://github.com/usabilityhub/rails-erb-loader/pull/78/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R54

**Comment by rhys-vdw** [comment]
Ah, I guess we should bump the minimum support.

**Comment by n-rodriguez** [comment]
Hi there! Any news?

**Comment by fiznool** [comment]
Will be fixed by #78

**Comment by rhys-vdw** [comment]
Guess this is closed.
